contracts/lib/BasicOrderFulfiller.sol

```diff
@@ -893,7 +888,10 @@ contract BasicOrderFulfiller is OrderValidator {
             mstore(ZeroSlot, 0)
 
             // Update the free memory pointer so that event data is persisted.
-            mstore(0x40, add(0x80, add(eventDataPtr, dataSize)))
+            mstore(
+                FreeMemoryPointerSlot,
+                add(BasicOrder_order_typeHash_ptr, add(eventDataPtr, dataSize))
```

Comment:
it does not address the issue that `4` extra slots have been used.

See:
- https://github.com/spearbit-audits/review-seaport-1.2/issues/108

Reply:
https://github.com/ProjectOpenSea/seaport/pull/819/commits/e0749919ccc414bbdc4adf70e299be5d15f2fad0